### ISSUE TITLE
2.4.3: Focus Order

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5148]
**Level:** A
**Principle:** Operable
**Result:** *pending*

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html)

### Audit Checklist
- [ ] Review previous Audit Result
- [ ] Review work identified in previous audit and list open issues.
- [ ] Check for any other relevant open issues.
- [ ] Retest open issues, are these issues still present?
- [ ] Test for any new issues that may have been introduced.
- [ ] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
 - link previous audit and summarise what was find and the outcome.

 ### Progress since last audit

 ### Audit Notes

 ### Work arising

**Comment by StephDriver** [comment]
## Manual Testing

I tested a representative sample of pages by tabbing through with a keyboard.   I found two issues, both on the article page, one on Material and one on OLH, the clean theme had no issues.

On a Material article, the article text settings are only reached after one has tabbed through the whole article. This is unhelpful as if a user wants different text settings, it would be before they read the article. 

On an OLH article, the close button for an enlarged figure popup is not focusable. See further discussion of this on #5143 about Keyboard traps. 

## Developer Notes
The placement of the article text settings in material is a restriction of the layout of that theme.  This is discussed further in the PR that introduced this feature #4796.
